RadeonRays/CMakeLists.txt

```diff
@@ -217,7 +217,14 @@ endif (RR_USE_VULKAN)
 target_compile_features(RadeonRays PRIVATE cxx_std_14)
 
 if (UNIX AND NOT APPLE)
+        
+    if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm" OR CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
```

Comment:
I made the -march-Native only for ARM cpus so that I dont cause the same issue that occurred last time we turned this flag on.